docs/_scripts/fix_gdocs2rst.py

```diff
@@ -0,0 +1,24 @@
+in_rubric = False
```

Comment:
```suggestion
# Script used to migrate gdocs into RST. 
# Historically used for the salary survey, but might be useful otherwise. 
in_rubric = False
```